.github/workflows/android-perf.yml

```diff
@@ -15,7 +15,7 @@ on:
         description: Target devices to run benchmark
         required: false
         type: string
-        default: false
+        default: samsung_galaxy_s2x
```

Comment:
Given that we are going to have preset pool of devices, I think it's better to use GH choice input here instead of a free-form string, https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs. `samsung_galaxy_s2x` will be the same of the pool so that we can map it to the AWS pool ID later

Reply:
Yeah, I like this idea better.

Reply:
Looked at the example. The `choice` type is suitable for logging because each option is mutual exclusive. I'm curious if `choice` type allows to choose multiple options from the list.

Reply:
Looks like it can't. Switch back to use string instead to preserve the ability of scheduling to run a model across multiple devices. FYI, if the string is invalid, it will fail early as shown in #4484

Reply:
Yeah, it can't.  I guess we can use a comma-separated string then, and handle the validation inside the job itself.

Reply:
Yes, pretty much what #4484 does. That we, we don't end up creating combinations of options. Though it's not super handy as users will have to enter the comma-separated string names precisely.